daprdocs/content/en/reference/components-reference/supported-state-stores/setup-redis.md

```diff
@@ -244,9 +244,9 @@ Consider the example from ["How-To: Query state"]({{< ref "howto-state-query-api
 {{< tabs "Self-Hosted" "Kubernetes" "Azure" "AWS" "GCP" "Redis Enterprise Cloud" "Alibaba Cloud" >}}
 
 {{% codetab %}}
-If you are using a self-hosted deployment of Dapr v1.7 and up, a Redis instance with the required modules is automatically created as a Docker container when you run `dapr init`.
+The Redis instance that gets created automatically with `dapr init` cannot be used with query api.
+Use following command to create an instance of redis which can be used with query api:
 
-Alternatively, you can create an instance of Redis by running the following command:
 ```bash
 docker run -p 6379:6379 --name redis --rm redislabs/rejson:2.0.6
```

Comment:
ask them to use a different port. 
Add in the component YAML file also ... .

Reply:
Can we make sure all the steps are there. 
1) Kill the current redis instance
2) Start the new instance with a different port just in case
3) Provide the YAML file for the query for redis.

Reply:
if a different port is used, the existing Redis instance can also run side by side.
+1 to providing a default YAML file to connect. Also add link to the query state api doc for the component file modification needed to add query index etc.

Reply:
An example of component file is already provided in following lines. check line#311